.github/CONTRIBUTING.md

```diff
@@ -1,47 +1,50 @@
 
 Our procedures for contributing bigger changes, code in particular, generally follow those advised by the tidyverse dev team, including following the tidyverse style guide for code and recording user facing changes in `NEWS.md`.
 
 ### Pull request process
 
-*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("hubverse-org/hubVis", fork = TRUE)`.
+- Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("hubverse-org/hubVis", fork = TRUE)`.
 
-*   Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`. 
-    If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing. 
-*   Create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("brief-description-of-change")`.
+- Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`.
+  If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing.
 
-*   Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser.
-    The title of your PR should briefly describe the change.
-    The body of your PR should contain `Fixes #issue-number`.
+- Follow [the pull request checklist](https://hubverse-org.github.io/hubDevs/articles/release-checklists.html#subsequent-pr-checklist) to create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("name/brief-description/issue")`.
 
-*  For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first header). Follow the style described in <https://style.tidyverse.org/news.html>.
+- Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser.
+  The title of your PR should briefly describe the change.
+  The body of your PR should contain `Fixes #issue-number`.
+
+- For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first heading---usually labelled "development version"). Follow the style described in <https://style.tidyverse.org/news.html>.
 
 ### Code style
 
-*   New code should follow the tidyverse [style guide](https://style.tidyverse.org). 
-    You can use the [styler](https://CRAN.R-project.org/package=styler) package to apply these styles, but please don't restyle code that has nothing to do with your PR.  
+- New code should follow the tidyverse [style guide](https://style.tidyverse.org).
+  You can use the [styler](https://CRAN.R-project.org/package=styler) package to apply these styles, but please don't restyle code that has nothing to do with your PR.
```

Comment:
Should we add "or lintr package" ? 
```suggestion
  You can use the [lintr](https://cran.r-project.org/web/packages/lintr/index.html) or [styler](https://CRAN.R-project.org/package=styler) package to apply these styles, but please don't restyle code that has nothing to do with your PR.
```

Reply:
Lintr is just going to complain if your code is not right, but stylr will actually change the code. We also use lintr in our CI workflows, so I will leave this as is. Thank you!